components/tools/OmeroJava/test/integration/gateway/RawDataFacilityTest.java

```diff
@@ -90,6 +91,26 @@ public void testGetTile() throws DataSourceException, DSOutOfServiceException, D
         Assert.assertEquals(planeData, rawDataPart);
     }
     
+    @Test
+    public void testGetHistogram() throws DataSourceException,
+            DSOutOfServiceException, DSAccessException {
+        ImageData img = browseFacility.getImage(rootCtx, imgId);
+
+        int[] exp = new int[256];
+        for (byte b : rawData) {
+            exp[Byte.toUnsignedInt(b)]++;
```

Comment:
it is Java 1.8 specific
no such method in Java 1.7 see https://docs.oracle.com/javase/7/docs/api/java/lang/Byte.html
and https://docs.oracle.com/javase/8/docs/api/java/lang/Byte.html